hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsStatistics.java

```diff
@@ -202,16 +182,13 @@ public void testOpenAppendRenameExists() throws IOException {
     fs.create(createFilePath);
     fs.open(createFilePath);
     fs.append(createFilePath);
-    fs.rename(createFilePath, destCreateFilePath);
+    assertTrue(fs.rename(createFilePath, destCreateFilePath));
```

Comment:
nit: There's a ContractTestUtils rename which is a bit more informative on rename failure, but I Don't see this failing enough to worry about it.

Reply:
I'll make sure to use that going forward.